src/librustc_codegen_llvm/intrinsic.rs

```diff
@@ -1171,6 +1171,24 @@ fn generic_simd_intrinsic(
     );
     let arg_tys = sig.inputs();
 
+    if name == "simd_select_bitmask" {
+        let in_ty = arg_tys[0];
+        let m_len = match in_ty.sty {
+            ty::Int(i) => i.bit_width().unwrap(),
+            ty::Uint(i) => i.bit_width().unwrap(),
+            _ => return_error!("`{}` is not an integral type", in_ty),
+        };
+        let v_len = arg_tys[1].simd_size(tcx);
```

Comment:
I believe there needs to be a `require_simd!` for the vector arguments (the second type parameter) to catch things like `simd_select_bitmask(0u8, "foo", "bar")`.

Reply:
Oh oops sure thing